shared/src/androidMain/kotlin/io.newm.shared/internal/implementations/PreferencesDataStoreImpl.android.kt

```diff
@@ -64,4 +64,12 @@ class PreferencesDataStoreImpl(private val context: Context) : PreferencesDataSt
             }
         }
     }
+
+    override fun clearAll() {
+        runBlocking {
```

Comment:
We are not able to pass the context into the method since we the interface is shared across android and iOS, but the Android implementation class does have access to the context so maybe I can use that instead. 
I don't think the `clearAll` needs to be suspended it as is a fire and forget type of operation. We use it when we log out the user.